test/benchgc/src/benchgc/util.py

```diff
@@ -131,16 +132,18 @@ def get_dtype(dtype: str) -> torch.dtype:
         return torch.bfloat16
     elif dtype == "u8" or dtype == "ui8":
         return torch.uint8
-    elif dtype == "s8" or dtype == "i8":
+    elif dtype in ["s8", "i8", "si8"]:
```

Comment:
What does `si8` and `si32` mean?

Reply:
signless -> i32/i8 
signed -> si32/si8
unsigned -> ui32/ui8